spec/models/emancipation_option_spec.rb

```diff
@@ -6,6 +6,12 @@
   it { is_expected.to have_many(:casa_cases).through(:casa_cases_emancipation_options) }
   it { is_expected.to validate_presence_of(:name) }
 
+  it "is unique across emancipation_category, name" do
+    eo = create(:emancipation_option)
+    eo_new = build(:emancipation_option, emancipation_category: eo.emancipation_category, name: eo.name)
+    expect(eo_new.valid?).to be false
+  end
+
```

Comment:
I think I already wrote this starting at line 21. One of the tests needs to go

Reply:
Ah, yes. You are testing the error at the database level, but now I've stopped the validation at the Rails level, so I'll remove the db checking one.